cashctrl_ledger/ledger.py

```diff
@@ -388,16 +399,22 @@ def add_ledger_entry(self, entry: pd.DataFrame):
                 raise ValueError('CashCtrl only allows for a single currency in a collective booking.')
             if entry['date'].nunique() != 1:
                 raise ValueError('Date should be the same in a collective booking.')
+            fx_rates = (entry['base_currency_amount'] / entry['amount']).abs()
+            # TODO: Find a more robust way to detect incoherent currency rates
+            if ((fx_rates - fx_rates.median()).abs() > 1e-8).any():
```

Comment:
Please add a test to check whether this error is raised if collective transactions have differing FX rates.